### DISCUSSION TITLE
Microsoft 365 domains (DNS)?

### DISCUSSION THREAD

**Discussion by plettb** [polymind-inc/acmebot#462] [Q&A]
Has anyone had any luck creating a wildcard against a Microsoft 365 DNS?  I'm not sure why, but Microsoft seems to have two different DNS services, Azure and Microsoft 365.  They don't seem to be interchangeable.

**Comment by shibayan** [comment]
The best option is to use Azure DNS; I don't know what Microsoft 365 DNS is like, but it would be difficult to support and maintain.

**Reply by plettb** [reply]
This doesn't answer the question, and doesn't account for the benefits provided by Microsoft 365 DNS.
